plugins/statistics.koplugin/main.lua

```diff
@@ -1410,6 +1429,21 @@ Time is in hours and minutes.]]),
     }
 end
 
+function ReaderStatistics:onUsePageLabelsUpdated()
+    self.use_pagemap_for_stats = self:usePageMapForStats()
+    local new_current_page
+    if self.document:hasHiddenFlows() and self.view.state.page then
+        return
+    end
+    if self.use_pagemap_for_stats then
```

Comment:
Maybe re-order these line to be more logical:
`local new_current_page` before it is used, after the `if` that would return and not need to have it.
May be a comment for the     `if self.document:hasHiddenFlows() and self.view.state.page then` (because just reading it now, I dunno (or have already forgotten) why we would do nothing at all :)